Gemfile

```diff
@@ -283,7 +283,9 @@ group :web_socket, :manageiq_default do
 end
 
 group :appliance, :optional => true do
+  gem "irb",                            "=1.4.1",            :require => false # Locked to same version as the installed RPM rubygem-irb-1.4.1-142.module_el9+787+b20bfeee.noarch so that we don't bundle our own
```

Comment:
Not sure if we want to lock down to this version...

Reply:
do we want to let the gemfile lock release file handle locking down the version?

Reply:
@bdunne I'm just waiting on unlocking of this line before merge.